client/packages/common/src/ui/components/inputs/Filters/FilterMenu.tsx

```diff
@@ -90,6 +85,21 @@ export const FilterMenu: FC<FilterDefinitions> = ({ filters }) => {
 
   const showRemoveOption = activeFilters.length > 0;
 
+  // updating active filters when the filters are changed
```

Comment:
as noted. required to have the filters update when they are changed, such as when new options are provided